src/components/Main/Main.js

```diff
@@ -0,0 +1,45 @@
+import { MDCDrawer } from "@material/drawer";
+
+import { PlayerComponent } from "../Player/Player";
+import { HeaderComponent } from "../Header/Header";
+import { SearchComponent } from "../Search/Search";
+import Main from "./Main.html";
+
+export class MainComponent {
+  constructor(mountPoint) {
+    this.mountPoint = mountPoint;
+  }
+
+  querySelectors() {
+    this.scrollTarget = document.getElementById("main-content");
```

Comment:
I am not sure, so I am only asking) It can be fully right way. Should you use this.mountPoint instead of document to get node elements? I mean `this.mountPoint.getElementById("main-content")` instead of `document.getElementById("main-content")`?

Reply:
Hi Eugene. Nope, this will not gonna work in this way.